lib/MailgunSend.js

```diff
@@ -68,23 +67,29 @@ class MailgunSend {
    */
   send(options) {
     return new Promise((fulfill, reject) => {
-      if (!MailgunSend.validate(options)) {
-        reject(new Error('Invalid options were supplied'));
+      try {
+        MailgunSend.validate(options);
+      } catch (e) {
+        reject(e);
+        return false;
```

Comment:
Can we refactor this to use async/await? It might clean up error handling, since it'll allow use one `catch` for both validation errors and `mailgun.messages.create` errors